src/components/header.tsx

```diff
@@ -140,54 +145,101 @@ export const Header = (props: Props) => {
                 >
+                  <SliderMark value={2} {...labelStyles}>
+                    slow
+                  </SliderMark>
+                  <SliderMark value={19} {...labelStyles}>
+                    fast
+                  </SliderMark>
                   <SliderTrack>
                     <SliderFilledTrack />
                   </SliderTrack>
                   <SliderThumb />
                 </Slider>
               </Box>
-              {/* <Box w="100%" pt={2}>
-                <Text fontSize={24} fontWeight="medium">
-                  Layer Transparency
-                </Text>
-                <Text fontSize={16} pb={2}>
-                  (only map)
-                </Text>
-                <Slider aria-label="slider-ex-2" colorScheme="teal" defaultValue={30}>
-                  <SliderTrack>
-                    <SliderFilledTrack />
-                  </SliderTrack>
-                  <SliderThumb />
-                </Slider>
-              </Box> */}
               <Divider borderColor="gray.600" pt={4} />
               <Box w="100%" py={2}>
                 <Text fontSize={24} fontWeight="medium" pb={2}>
-                  Layer Select
+                  Layer Settings
                 </Text>
-                <RadioGroup colorScheme="teal">
+                <InfoBox>Select the second layer for the 2D map.</InfoBox>
+                <Box w="100%">
+                  <SwitchSetting
+                    name="Display Layer"
+                    checked={option.displayLayer}
+                    onChange={() => setOption({ ...option, displayLayer: !option.displayLayer })}
+                  />
+                  <Divider borderColor="gray.400" my={4} />
+                  <Box w="100%" pb={2}>
+                    <Text fontSize={16} pb={2}>
+                      Opacity: {option.layerOpacity * 100}%
+                    </Text>
+                    <Slider
+                      aria-label="slider-ex-3"
+                      colorScheme="teal"
+                      min={0}
+                      max={100}
+                      step={10}
+                      defaultValue={option.layerOpacity * 100}
+                      onChange={(e) => setOption({ ...option, layerOpacity: e / 100 })}
+                    >
+                      <SliderMark value={7} {...labelStyles}>
+                        transparent
+                      </SliderMark>
+                      <SliderMark value={84} {...labelStyles}>
+                        opaque
+                      </SliderMark>
+                      <SliderTrack>
+                        <SliderFilledTrack />
+                      </SliderTrack>
+                      <SliderThumb />
+                    </Slider>
+                  </Box>
+                </Box>
+                <Divider borderColor="gray.400" my={4} />
+                <RadioGroup
+                  colorScheme="teal"
+                  defaultValue="0"
```

Comment:
選択レイヤーがメニュー開閉時に毎回初期化されて嫌そう
```suggestion
                  defaultValue={option.layerIdx.toString()}
```

Reply:
これ変え忘れだね
反映しました〜　https://github.com/yudai1204/nasa-hackathon-2023-yokohama/pull/101/commits/5696c672d97a407a33fe74906ae536452b29d62c